src/src/Commands/OpenCommand.php

```diff
@@ -2,27 +2,27 @@
 
 namespace QIT_CLI\Commands;
 
-use Symfony\Component\Console\Command\Command;
 use Symfony\Component\Console\Input\ArrayInput;
 use Symfony\Component\Console\Input\InputArgument;
 use Symfony\Component\Console\Input\InputInterface;
 use Symfony\Component\Console\Output\OutputInterface;
 
-class OpenCommand extends Command {
+class OpenCommand extends QITCommand {
 	protected static $defaultName = 'open'; // phpcs:ignore WordPress.NamingConventions.ValidVariableName.PropertyNotSnakeCase
 
-	protected function configure() {
+	protected function configure(): void {
+		parent::configure();
 		$this
 			->setDescription( 'Open a test run result in the browser.' )
 			->addArgument( 'test_run_id', InputArgument::REQUIRED );
 	}
 
-	protected function execute( InputInterface $input, OutputInterface $output ): int {
+	protected function doExecute( InputInterface $input, OutputInterface $output ): int {
 		$command = $this->getApplication()->find( GetCommand::getDefaultName() );
-
-		return $command->run( new ArrayInput( [
+		$command->run( new ArrayInput( [
```

Comment:
Same as above, we used to run a command and now we return SUCCESS. Is this expected? Also, what if the command fails? It would stop the execution and we wouldn't return SUCCESS here, right?